doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,133 @@
+               should be held to ensure proper review. See
+               :ref:`review process <review_time>`
+=============  ===============================================================
+
+Issue priority labels
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the impact and importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are generally set or changed during the bug-triage or TSC
+meetings.
+
+Miscellaneous labels
+********************
+
+For both PRs and issues
+=======================
+
++------------------------+-----------------------------------------------------+
+|``Bug``                 | The issue is a bug, or the PR is fixing a bug       |
++------------------------+-----------------------------------------------------+
+|``Coverity``            | A Coverity detected issue or its fix                |
++------------------------+-----------------------------------------------------+
+|``Waiting for response``| The Zephyr developers are waiting for the submitter |
+|                        | to respond to a question, or address an issue.      |
++------------------------+-----------------------------------------------------+
+|``RFC``                 | The author would like input from the community. For |
+|                        | a PR it should be considered a draft                |
++------------------------+-----------------------------------------------------+
+|``LTS``                 | Long term release branch related                    |
++------------------------+-----------------------------------------------------+
+|``In progress``         | For PRs: is work in progress and should not be      |
+|                        | merged yet. For issues: Is being worked on          |
++------------------------+-----------------------------------------------------+
+|``EXT``                 | Related to an external component (in ``ext/``)      |
++------------------------+-----------------------------------------------------+
+
+PR only labels
+==============
+
+================ ===============================================================
+``DNM``          This PR should not be merged (Do Not Merge).
+                 For work in progress, GitHub "draft" PRs are preferred
+``Stale PR``     PR which seems abandoned, and requires attention by the author
+``Blocked``      Blocked by another PR or issue
+``needs review`` The PR needs attention from the maintainers
+``Backport``     The PR is a backport or should be backported
+``Licensing``    The PR has licensing issues which require a licensing expert to
+                 review it
+================ ===============================================================
+
+Issue only labels
+=================
```

Comment:
I think it's important to note that an issue needs to have one of the 4 following labels:

- Bug
- Question
- Enhancement
- Feature Request

Reply:
The only way to make that clear and simple is for them to share a common prefix, I suggest: `Type: bug`, `Type: Question`,... This will also group them in the list.